web/__tests__/pages/api/post/post.int.test.ts

```diff
@@ -0,0 +1,154 @@
+import { readFileSync } from "fs";
+import { createRandomPost } from "../../../../db/actions/__mocks__/Post";
+import { consts } from "../../../../utils/consts";
+import storageClient from "../../../../db/storageConnect";
+
+const noResizeData = readFileSync("./__tests__/assets/no-resize.png");
+const resizeData = readFileSync("./__tests__/assets/resize.png");
+const videoData = readFileSync("./__tests__/assets/video.mp4");
+
+jest.setTimeout(10000);
```

Comment:
The timeout seems to have caused the build error because the logs say it's a problem with `jest`, so I'll remove that.

Reply:
That doesn't seem to have fixed it, that's strange.